torch/jit/_script.py

```diff
@@ -984,6 +988,12 @@ def forward(self, input):
         _set_jit_function_cache(obj, fn)
         return fn
 
+    raise RuntimeError(
+        "Unsupported torch.jit.script target: \"{}\", "
+        "only classes, functions and "
+        "instances of 'nn.Module' are allowed.".format(obj)
```

Comment:
I think we should get in the habit of using f-strings now that Python2 support has been dropped.